src/frontend/useOnBackgroundedAndForegrounded.ts

```diff
@@ -0,0 +1,16 @@
+import {type MapeoClientApi} from '@mapeo/ipc';
+import {useEffect} from 'react';
+import {useAppState} from '@react-native-community/hooks';
+
+export default function useOnBackgroundedAndForegrounded(
```

Comment:
Also, we stay away from default exports in the front end (because we are civilized people...)

Reply:
🔥 🔥 🔥